packages/modules/psa/soc.py

```diff
@@ -0,0 +1,60 @@
+from typing import Union, List
+
+import logging
+
+from dataclass_utils import dataclass_from_dict
+from helpermodules.cli import run_using_positional_cli_args
+from modules.common import store
+from modules.common.abstract_device import DeviceDescriptor
+from modules.common.abstract_soc import AbstractSoc
+from modules.common.component_context import SingleComponentUpdateContext
+from modules.common.component_state import CarState
+from modules.common.fault_state import ComponentInfo
+from modules.psa import api
+from modules.psa.config import PSA, PSAConfiguration
+
+
+log = logging.getLogger("soc."+__name__)
```

Comment:
Eine Logausgabe sieht dann zum Beispiel so aus:
```python
2022-11-21 13:06:31,761 - {soc.modules.psa.soc:10} - {WARNING:MainThread} - Hi there
```
Fokus auf den Loggername: `soc.modules.psa.soc`. Das `soc` ist so doppelt. Es kommt einmal von `__name__`, den die Datei heißt `soc.py`.

Ich finde es konsequenter nur bei `__name__` zu bleiben, denn da steckt alles drin. Wenn die Initialisierung aus einer anderen Datei kommt, zum Beispiel aus der `api.py`, dann hieße der Logger `modules.psa.api`, was soc nicht enthält, die Quelle aber trotzdem eindeutig identifiziert.

 Ich sehe allerdings, dass auch einige weitere Module das Konstrukt so haben. Falls das eine Designentscheidung so ist müsste das natürlich so bleiben...

Reply:
Ja, die anderen Module machen das auch genau so, ggf. kann man das zukünftig in allen Modulen ändern wenn sich das Design ändert.